src/main/kotlin/com/team4099/robot2022/subsystems/Wheel.kt

```diff
@@ -233,7 +230,7 @@ class Wheel(
           speed
         }
     directionSetPoint = directionSensor.position + directionDifference
-    driveSpark.set(outputPower)
+    driveFalcon.set(ControlMode.PercentOutput, outputPower)
   }
 
   fun resetModuleZero() {
```

Comment:
ok can we change this to just print the absolute encoder value without the offset in the logger line? (so like instead of using the `AngularMechanismSensor` like usual we would use the `AnalogPotentiometer` directly). this would let us basically manually line up all the modules, see what it printed, then adjust the zero offset in the drivetrain code accordingly